src/server/pingserver-rs/src/admin.rs

```diff
@@ -14,13 +14,14 @@
 // limitations under the License.
 
 use crate::stats::Metrics;
+
 use ccommon::metric::MetricExt;
-use pelikan::core::admin::AdminHandler;
 use pelikan::protocol::{admin::AdminProtocol, Protocol};
 use pelikan_sys::protocol::admin::{
-    METRIC_END_LEN, METRIC_PRINT_LEN, REQ_STATS, REQ_VERSION, RSP_GENERIC, RSP_INVALID,
+    METRIC_END_LEN, METRIC_PRINT_LEN, REQ_QUIT, REQ_STATS, REQ_VERSION, RSP_GENERIC, RSP_INVALID,
```

Comment:
Do we want to just do a wildcard import here?

Reply:
Done.

Reply:
Done.